File: no 2/cpp/product.cpp
class Product{
	private:
		string price;
		string id_product;

	public:
		Product(){
		}

		// set dan get dari private
		void setPrice(string price){
			this->price = price;
		}
		string getPrice(){
			return this->price;
		}

		void setId_Product(string id_product){
			this->id_product = id_product;
		}
		string getId_Product(){
			return this->id_product;
		}

		~Product(){
		}

};

File: no 2/php/product.php
<?php

class Product{
	private $price;
	private $id_product;

	// set dan get 

	function __construct(){
	}

	function setPrice($price){
		$this->price = $price;
	}
	function getPrice(){
		return $this->price;
	}

	function setId_Product($id_product){
		$this->id_product = $id_product;
	}
	function getId_Product(){
		return $this->id_product;
	}

	function __destruct(){

	}

}

?>